backend/src/main/kotlin/outcobra/server/model/dto/MarkGroupDto.kt

```diff
@@ -20,7 +20,7 @@ data class MarkGroupDto(override val id: Long = 0,
                         val markGroups: List<MarkGroupDto> = listOf()) : BaseMarkDto {
     override val parentLink: ParentLink
         get() {
-            if (!(subjectId == 0L).xor(parentGroupId == 0L)) {
+            if (subjectId == 0L && parentGroupId == 0L) {
```

Comment:
There is no reason why a dto should have two different parent objects in my opinion...

Reply:
Imho it makes sense how it is now (with the change) and it makes the frontend easier bc the xor introduced a bug which wasn't noticed until now.

Reply:
There is a slight possibility that this is going to cause issues within the request / child validation.
Did you check that? Also can you please explain the bug a little further?

Reply:
Request validation works.
The subjectId was 0 and the frontend couldn't open the correct subject when coming from an create or update view.

Reply:
You can see it on staging.